include/image.mk

```diff
@@ -234,13 +234,26 @@ endef
 $(eval $(foreach S,$(JFFS2_BLOCKSIZE),$(call Image/mkfs/jffs2/template,$(S))))
 $(eval $(foreach S,$(NAND_BLOCKSIZE),$(call Image/mkfs/jffs2-nand/template,$(S))))
 
-define Image/mkfs/squashfs
+define Image/mkfs/squashfs-common
 	$(STAGING_DIR_HOST)/bin/mksquashfs4 $(call mkfs_target_dir,$(1)) $@ \
 		-nopad -noappend -root-owned \
 		-comp $(SQUASHFSCOMP) $(SQUASHFSOPT) \
 		-processors 1
 endef
 
+ifeq ($(CONFIG_PACKAGE_refpolicy),y)
```

Comment:
This will not work for our binary builds having `CONFIG_ALL=y` and hence also `CONFIG_PACKAGE_refpolicy=y` always set.

Reply:
I changed this to
```
ifeq ($(CONFIG_KERNEL_SECURITY_SELINUX),y)
```
Does this solve the problem?

Reply:
I like Paul's suggestion of introducing an independent config symbol like you did in your most recent update of the PR.